py/dml/structure.py

```diff
@@ -185,9 +185,11 @@ def mkglobals(stmts):
                 new_typedefs.add(name)
             elif stmt[0] == 'loggroup':
                 _, site, name = stmt
+                if len(dml.globals.log_groups) >= 63:
+                    report(ELOGGROUPS(site))
                 dml.globals.log_groups.append(name)
                 new_symbols.append(LiteralSymbol(
-                    name, TNamed('int', const=True), site,
+                    name, TInt(64, False, const=True), site,
```

Comment:
I think this is a real bugfix, probably possible to capture with some nonsensical expression like `lg32 < 0`, but probably not worth adding a test.